.travis.yml

```diff
@@ -1,11 +1,36 @@
-language: minimal
+language: generic
 
-install:
-  - ./build.sh
+matrix:
+  include:
+    - os: linux
+      env:
+        - DOCKERIMAGE=condaforge/linux-anvil-aarch64
+    # - os: linux
+    #   env:
+    #     - DOCKERIMAGE=condaforge/linux-anvil-ppc64le
+    # - os: linux
+    #   env:
+    #     - DOCKERIMAGE=condaforge/linux-anvil-comp7
+
+before_install:
+  # Enable experimental features (needed for squash)
+  - sudo cp docker_daemon_config.json /etc/docker/daemon.json
+  - sudo service docker restart
+
+  - docker info
+  - docker run --rm --privileged multiarch/qemu-user-static:register --reset
+
+script:
+  - echo "* Fix permissions"
+  - chmod 777 build/
+  - echo "* Build Miniforge installer"
+  - docker run --rm -ti --mount type=bind,source="$(pwd)",target=/construct ${DOCKERIMAGE} /construct/build.sh
+  - echo "* Test Miniforge installer"
+  - docker run --rm -ti --mount type=bind,source="$(pwd)",target=/construct ${DOCKERIMAGE} /construct/test.sh
```

Comment:
I don't think testing should be done the linux anvil image. It is very unlikely that users will use it. Does it make sense to test on a bare cos7 or ubuntu 16.04 image?

Reply:
True. If we fixed https://github.com/conda-forge/docker-images/issues/122, then we can use the scripts there to build that image and use the testing scripts from there as well.

Reply:
I mean that can be left for a future issue. just wanted to bring it up.

Reply:
Actually there is no need to use conda image. `test.sh` can use `arm64v8/centos:7`.

Reply:
See https://github.com/conda-forge/miniforge/pull/2/commits/21d417ee9fc3651b378c3a0d10e9c8ee046059a7 and let me know what do you think.